docs/source/logging/logging.md

```diff
@@ -25,7 +25,7 @@ In addition to the `rich` handler defined in Kedro's framework, the [project-sid
 
 The logging handlers that are actually used by default are `rich`, `info_file_handler` and `error_file_handler`.
 
-The project-side logging configuration also ensures that [logs emitted from your project's logger](#use-logging) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`).
+The project-side logging configuration also ensures that [logs emitted from your project's logger](#Perform logging in your project) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`).
```

Comment:
This looks a bit weird to me, could you check if this link is pointing to the right section? I think it usually automatically converts to lower case and trims the whitespace to something like `#perform-logging-in-your-project`.